e2e/tests/integrations/integrator/not-allowed-on-admin-pages.test.ts

```diff
@@ -6,5 +6,5 @@ test("Integrator access to Admin Page - Expect 403", async ({ page }) => {
   const response = await page.goto("/admin");
   const status = response?.status();
   expect(status).toBe(403);
-  await expect(page.getByText("403")).toBeVisible();
+  await expect(page.locator('h1').getByText("403")).toBeVisible();
```

Comment:
You can also be a bit less specific:

`page.getByRole('heading', { name: "403" })`